docs/internals.rst

```diff
@@ -1656,6 +1656,40 @@ significant variance. Experience shows that a ~20% time difference can be
 reliably measured on GitHub hosted runners, and smaller differences are
 noticeable over a few days of reruns as trends emerge from the noise.
 
+Code coverage
+-------------
+
+Code coverage for Verilator itself can be collected using GCC with the
+following flow. Note that configuring with ``--enable-coverage`` disables
+optimization for both the debug and optimized builds of Verilator, so running
+the resulting executables can be slow:
+
+.. code:: shell
+
+   ./configure --enable-longtests --enable-coverage
+   make -j$(nproc)        # Build verilator
+   make test              # Run the tests - this will generate .gcda files
+   make coverage-open     # Create and open HTML coverate reprot
+
+
+The ``coverage-open`` make target depends on ``covarage-report``, which
+generates the HTML coverage report, and does in turn depend on
+``coverage-combine``, which combines all ``.gcda`` files into an lcov
+data file. You can use each of these make targets separately if you wish.
```

Comment:
Coverage-open suggests it's starting coverage not viewing it.

Awkward, maybe:  (Avoid "you" - don't know who reader is.)

```suggestion
The ``coverage-view`` make target opens the generated coverage report. This depends on ``coverage-report``, which
generates the HTML coverage report. That in turn depends on
``coverage-combine``, which combines all ``.gcda`` files into an lcov
data file. Each of these make targets can be used separately if desired.
```

Reply:
> Awkward

I know, I am, sorry ;) Fixed.